plugin.json

```diff
@@ -2,6 +2,9 @@
     "id": "github",
     "name": "GitHub",
     "description": "GitHub plugin for Mattermost.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-github",
```

Comment:
These fields are modelled in https://github.com/mattermost/mattermost-plugin-github/pull/145, but it doesn't hurt to add them early. I have similar PRs prepared for other plugins once I prove that the generator works with a new release of this one.